### PR TITLE
Add git status -uno to the list of git commands

### PR DESCRIPTION
**Pull request by freeranger**
I don't have direct access to the repo yet to update this myself, so here is a pull reques.t
Just adding some git help to check they are up to date, as this is the second time in two days I've had to give a student this instruction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ git pull|Pull down any changes that were checked into a branch (Git repository).
 git push|Push your committed changes to your currently checked out branch.
 git push origin master|Push your committed changes up to your master branch. 'master' may be subsituted for another branch.
 git status|Get the status of files in the current repo.
+git status -uno|Check if your local branch is up-to-date with the remote one
```
Comment (jasonwr) [inline README.md]: @freeranger I would not recommend "git status -uno" because if you add a new file then this will not get picked up in the status change. "git status -uno" is the same as:

git status --untracked-files=no

Or:

git config status.showuntrackedfiles no

This is not good! Typically when you run status you want to see any new untracked files that need to be added. I would recommend just running a simple "git status" as this will reveal whether or not your local branch is up to date with the remote.

Comment (ychaikin) [inline README.md]: @freeranger @jasonwr Well, since this is for the beginner students and this is not a 'git' class, I want to stay away from adding more commands that are not directly related to what they need to do in the class.

**Comment by freeranger** [comment]
@jasonwr - the problem the students had was they could see files in GitHub but not in their local repo so this enables them to check without the "noise" of additional untracked files.

@ychaikin I was adding it because a couple of students had problems with their local vs remote repos which they needed to resolve for the course, so in that sense it is something potentially "needed" for the course, but I understand there has to be a line in the sand or else you end up with everything to do with every possibility in your docs.

I wonder if this is the model to adopt - rather than us having rights to update your repos, w resubmit PRs for review and you accept or not as you seem appropriate - this means you always automatically know about any changes to the docs/FAQs because you get a PR notification, and other people can comment on them too, as has happened in this case.

**Comment by jasonwr** [comment]
@freeranger well you shouldn't need a local repo. Typically what I do is clone the master repo then branch off of that. Easy peezy.

**Comment by freeranger** [comment]
@jasonwr - you can't help but have a local repo - you clone the master, that clone is a local copy of the entire repo.

**Comment by jasonwr** [comment]
@freeranger yes but your repo is still associated with a remote repository (git push origin <branch>). "git status" is preferable because "git status -uno" will not report on untracked files and this can be deceiving because any new files will be ignored.

**Comment by ychaikin** [comment]
@freeranger I think there is a miscommunication going on here and I am no longer sure what @freeranger is proposing and why. Can you please email the Google group with a more detailed explanation of why you think students need 'git status -uno' as part of their arsenal of git commands?

As far as PRs, I am fine with that. Just realize that the Wiki is a separate repo from the code repo.
